llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2374,6 +2380,44 @@ class VPReductionPHIRecipe : public VPHeaderPHIRecipe,
   bool isInLoop() const { return IsInLoop; }
 };
 
+/// A recipe for forming partial reductions. In the loop, an accumulator and
+/// vector operand are added together and passed to the next iteration as the
+/// next accumulator. After the loop body, the accumulator is reduced to a
+/// scalar value.
+class VPPartialReductionRecipe : public VPRecipeWithIRFlags {
```

Comment:
Now that flag handling has been removed, can we directly inherit from VPSingleDefRecipe?